UpdateDependencies.ps1

```diff
@@ -10,30 +10,37 @@ param(
     [Parameter(Mandatory=$true)][string]$GitHubUser,
     [Parameter(Mandatory=$true)][string]$GitHubEmail,
     [Parameter(Mandatory=$true)][string]$GitHubPassword,
-    [Parameter(Mandatory=$true)][string]$CoreFxVersionUrl, 
+    [Parameter(Mandatory=$true)][string]$CoreFxVersionUrl,
+    [string[]]$DirPropsVersionElements = 'CoreFxExpectedPrerelease',
     [string]$GitHubUpstreamOwner='dotnet', 
     [string]$GitHubOriginOwner=$GitHubUser,
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
     [string]$GitHubPullRequestNotifications='')
 
-$CoreFxLatestVersion = Invoke-WebRequest $CoreFxVersionUrl -UseBasicParsing
-$CoreFxLatestVersion = $CoreFxLatestVersion.ToString().Trim()
+$LatestVersion = Invoke-WebRequest $CoreFxVersionUrl -UseBasicParsing
```

Comment:
If not, I'd assume you should change the $CoreFxVersionUrl parameter to be $LatestVersionUrl, or something similarly named.

---
In reply to: [65294139](https://github.com/dotnet/corefx/pull/8999#discussion_r65294139) [](ancestors = 65294139)